src/utils/convertCSVToJSON.ts

```diff
@@ -39,6 +40,7 @@ export const convertCSVToJSON: (string) => Promise<Collection[]> = (
                 headerImage,
                 credit,
                 price_guidance,
+                show_on_editorial,
```

Comment:
we should add default value to `false` here & make sure we test ingestion script

Reply:
Thanks for the reminder, this is now updated and tests as well.